src/io/flutter/preview/render_server_template.txt

```diff
@@ -181,8 +181,8 @@ class _DesignerApp extends StatelessWidget {
       home: new Scaffold(
         body: new Center(
           child: new SizedBox(
-            width: 350.0,
-            height: 400.0,
+            width: 350.0 /*TODO: width*/,
```

Comment:
this code would become
width: __WINDOW_WIDTH__,
or something along those lines with my suggested change.
/*TODO: width*/ makes it look like this a TODO suggesting that we should pass in the width not
the way you are actually passing in the width.